flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/SchemaTranslatorTest.java

```diff
@@ -203,31 +207,31 @@ public void testInputFromRowWithPhysicalDeclaredSchema() {
                                 .columnByExpression("computed2", "f1 - 1")
                                 .build());
 
-        assertEquals(
-                DataTypes.ROW(
-                                DataTypes.FIELD("f0", DataTypes.INT()),
-                                DataTypes.FIELD("f1", DataTypes.BIGINT()),
-                                DataTypes.FIELD("f2", DataTypes.DECIMAL(10, 2)),
-                                DataTypes.FIELD("f3", DataTypes.BOOLEAN()))
-                        .notNull(),
-                result.getPhysicalDataType());
-
-        assertTrue(result.isTopLevelRecord());
-
-        assertEquals(
-                Schema.newBuilder()
-                        .column("f0", DataTypes.INT())
-                        .column("f1", DataTypes.BIGINT())
-                        .column("f2", DataTypes.DECIMAL(10, 2))
-                        .column("f3", DataTypes.BOOLEAN())
-                        .columnByExpression("computed", "f1 + 42")
-                        .columnByExpression("computed2", "f1 - 1")
-                        .primaryKeyNamed("pk", "f0")
-                        .build(),
-                result.getSchema());
-
-        assertEquals(
-                Arrays.asList("f1", "f0", "computed", "f2", "computed2"), result.getProjections());
+        assertThat(result.getPhysicalDataType())
+                .isEqualTo(
+                        DataTypes.ROW(
```

Comment:
Static import for `ROW`, `FIELD` and other `DataTypes` methods?

Reply:
done